physionet-django/user/validators.py

```diff
@@ -255,3 +256,11 @@ def validate_domain_list(value):
     """
     if not re.fullmatch(r'(\w+\.\w+,*\s*)*', value):
         raise ValidationError('Must be separated with commas.')
+
+
+def validate_training_file_size(value):
+    """
+    Validate the file size of a file.
+    """
+    if value.size > settings.MAX_TRAINING_REPORT_UPLOAD_SIZE_MB * 1024 * 1024:
+        raise ValidationError('The maximum file size that can be uploaded is 5MB.')
```

Comment:
I don't think we want to hardcode `5MB` here. I guess we may want to do the byte to KB/MB conversion though? 

If we are rounding, we need to avoid small numbers being rounded to 0. @bemoody copying you in in case you have idea about how best to deal with this? Could be an argument for requiring the file size to be specified in KB/MB?

Reply:
Ah yes, my bad. Thank you.

Benjamin mention that we should use KB earlier. i had that changes but it seems i lost it when pushing here.

Doing that asap

Reply:
i see you meant to talk about rounding it down.

It should be a perfect round number  i think as our predefined size limit is a 1048576 Bytes, which should be 1024 KB.

So even if we update the limit in future, we should be fine as long a we use a whole number limit like 1 MB, 2 MB etc.

Reply:
pushed the changes, but not marking this as resolved as we are waiting to hear from Benjamin